models/public/ctdet_coco_dlav0_384-onnx/ctdet_coco_dlav0_384-onnx.md

```diff
@@ -0,0 +1,130 @@
+# ctdet_coco_dlav0_384-onnx
+
+## Use Case and High-Level Description
+
+CenterNet object detection model `ctdet_coco_dlav0_384-onnx` originally trained on PyTorch\*
+than converted to ONNX\* format. CenterNet model an object as a single point - the center point of its bounding box
+and uses keypoint estimation to find center points and regresses to object size.
+For details see [paper](http://arxiv.org/abs/1904.07850), [repository](https://github.com/xingyizhou/CenterNet/).
+
+### Steps to Reproduce PyTorch to ONNX Conversion
+By default model is provided in ONNX format, which was obtained by the following steps.
+
+1. Clone the original repository
+```sh
+git clone https://github.com/xingyizhou/CenterNet
+cd CenterNet
+```
+2. (Optional) Checkout the commit that the conversion was tested on:
```

Comment:
"Optional" doesn't make sense, since this describes steps that already happened. :-)